staff/alex-martinez/precourse/airlines/airlinesPRO/index.js

```diff
@@ -0,0 +1,195 @@
+    { id: 04, to: 'Roma', from: 'Barcelona', cost: 150, scale: false },
+    { id: 05, to: 'London', from: 'Madrid', cost: 200, scale: false },
+    { id: 06, to: 'Madrid', from: 'Barcelona', cost: 90, scale: false },
+    { id: 07, to: 'Tokyo', from: 'Madrid', cost: 1500, scale: true },
+    { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+    { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+    { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false }
+];
+
+//constructor objeto Vuelo
+function Vuelo(id,to,from,cost,scale){
+    this.id = id;
+    this.to = to;
+    this.from = from;
+    this.cost = cost;
+    this.scale = scale;
+}
+
+//función para poner la primera letra del usuario en mayúscula
+function PrimeraMayus(string){
+    return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+//funcion inicial inicio sesión
+function saludo(){
+    //pasamos el nombre de usuario a minuscula + eliminar espacios
+    var usuario = prompt('Escibe tu usuario').toLowerCase().trim();
+    
+
+    if(usuario){
+        console.log(`Bienvenido ${PrimeraMayus(usuario)}`);
+    }
+}
+//función para mostrar los vuelos
+function vuelos(){
+    console.log('Vuelos de hoy: ')
+    for(var i = 0; i<flights.length; i++){
+        if(flights[i].scale === false){
+            var escala = 'no realiza ninguna escala';
+        }else{
+            escala = 'si realiza escala';
+        }
+
+        console.log(`El vuelo con origen ${flights[i].from} y destino ${flights[i].to} tiene un coste de ${flights[i].cost}€, y ${escala}`);
+    }
+}
+
+//función coste medio
+function costeMedioVuelos(){
+    //variable que sumará los precios de los vuelos
+    var costeTotal = 0;
+    //variable que almacenará el coste medio
+    var costeMedio = 0;
+    for(var i = 0; i<flights.length; i++){
+        //variable para acumular todos los costes
+        var costes = flights[i].cost;
+        //suma de los precios
+        costeTotal += costes;
+    }
+    //coste total / numero de total de vuelos, con dos decimales
+    costeMedio = parseFloat(costeTotal/flights.length).toFixed(2);
+    console.log(`El coste medio de los vuelos es de ${costeMedio}€`);
+}
+
+function vuelosEscalas(){
+    //array para guardar los vuelos con escala y despues contarlos
+    var conEscala = [];
+    console.log('Vuelos con escala: ')
+    for(var i=0; i<flights.length; i++){
+        if(flights[i].scale === true){
+            conEscala.push(flights[i]);
+            console.log(`Vuelo con origen ${flights[i].from} y destino ${flights[i].to}`);   
+        }    
+    }
+    console.log(`En total hay ${conEscala.length} vuelos con escala`);
+}
+
+//función para mostrar los 5 ultimos vuelos
+function ultimosCinco(){
+    //variable que almacena los 5 ultimos vuelos del array
+    var ultimosVuelos = flights.slice(flights.length-5);
+    console.log('Los últimos 5 vuelos del día son: ')
```

Comment:
Hey Alex, I suggest you to put ";" here to have all homogeneous :)

Reply:
I omitted this, thanks Marc